### PR TITLE
Delete Organization

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR allows the Organization administrators to delete the Organization

deleteOrg

#### Bonus
TSR: Show empty stars 

## Related issue/s
closes https://github.com/ITISFoundation/osparc-issues/issues/422

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]
great addition!

**Review by GitHK — APPROVED** [review]
👍

**Review by KZzizzle — APPROVED** [review]
amazing!

File: services/web/client/source/class/osparc/desktop/preferences/pages/OrganizationsPage.js [hunk 1]
```diff
@@ -275,6 +281,49 @@ qx.Class.define("osparc.desktop.preferences.pages.OrganizationsPage", {
...
+      win.addListener("close", () => {
+        if (win.getConfirmed()) {
+          const params = {
+            url: {
+              "gid": orgKey
+            }
+          };
+          osparc.data.Resources.fetch("organizations", "delete", params)
+            .then(() => {
+              osparc.store.Store.getInstance().reset("organizations");
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/desktop/preferences/pages/OrganizationsPage.js:309]: how does this affect all relations to an organization? e.g. classifiers, organization users, etc

Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/preferences/pages/OrganizationsPage.js:309]: Those entries in the DB that have the group id as a field are deleted. On the other hand, those other fields where the gid is part of a json object, I'm afraid that will stay orphan.

As you know, the DB requires some love, and once we refactor these kind of entries we will remove the orphan ones.

Comment (pcrespov) [reply services/web/client/source/class/osparc/desktop/preferences/pages/OrganizationsPage.js:309]: @odeimaiz can you then create a maintenance case to remind us to refaactor/review this please?

**Pull request merged**